### PR TITLE
Add test for tf-job-simple prototype

### PR DESCRIPTION
**Pull request by ankushagarwal**
tf_job_simple_test.py tests the tf-job-simple prototype by creating a tfjob from that prototype and checking that the appropriate pods and service accounts get created

Fixes https://github.com/kubeflow/kubeflow/issues/462

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1010)

### CODE REVIEW

**Comment by ankushagarwal** [comment]
/cc @lluunn @kunmingg 

This is ready for review

File: testing/tf_job_simple_test.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    default="",
+    type=str,
+    help="The kubeflow src directory")
+  args, _ = parser.parse_known_args()
+  return args
+
+@retry(wait_fixed=5000, stop_max_attempt_number=20)
+def wait_for_tf_job():
+  out = util.run(["kubectl", "get", "pods", "-l",
+                  "tf_job_name=mycnnjob", "-ndefault"])
```
Comment (lluunn) [inline testing/tf_job_simple_test.py]: If namespace is default, do we need to delete them? I guess jobs will gone after finish, how about the service?

Comment (ankushagarwal) [reply testing/tf_job_simple_test.py]: We will be deleting the cluster at the end so I did not bother deleting this.

Comment (lluunn) [reply testing/tf_job_simple_test.py]: Are we using kubeflow-testing cluster in kubeflow-ci?

Comment (ankushagarwal) [reply testing/tf_job_simple_test.py]: No, we'll be creating a new cluster for every test

**Pull request merged**